pandas/tests/io/excel/test_readers.py

```diff
@@ -433,9 +433,14 @@ def test_reader_special_dtypes(self, request, read_ext):
         float_expected = expected.copy()
         float_expected["IntCol"] = float_expected["IntCol"].astype(float)
         float_expected.loc[float_expected.index[1], "Str2Col"] = 3.0
-        actual = pd.read_excel(
-            basename + read_ext, sheet_name="Sheet1", convert_float=False
-        )
+        with tm.assert_produces_warning(
+            FutureWarning,
+            match="convert_float is deprecated",
+            raise_on_extra_warnings=False,
+        ):
+            actual = pd.read_excel(
```

Comment:
Comment here as well.

Reply:
Oops, missed that one. Thanks for catching it.